### PR TITLE
Explain how to disable editable placeholders

### PR DESCRIPTION
**Pull request by JakeSCahill**

### CODE REVIEW

**Comment by Deflaimun** [comment]
Maybe worth adding that it only works with code blocks and not code fences.

File: meta-docs/STYLE-GUIDE.adoc [hunk 1]
```diff
@@ -522,7 +522,7 @@ Avoid mentioning any future features or releases within the documentation. Such
 
 == Placeholders
 
-Placeholders in sample code and commands represent values that the user must replace. Placeholders in example output can also represent other values that vary. A placeholder has a descriptive name as its value. Separate words with a dash. *Do not* use possessives or instructions as values, such as replace-with or my-value.
+Placeholders in sample code and commands represent values that the user must replace. Placeholders in example output can also represent other values that vary. A placeholder has a descriptive name as its value. Always separate words with a dash. *Do not* use possessives or instructions as values, such as replace-with or my-value.
```
Comment (JakeSCahill) [inline meta-docs/STYLE-GUIDE.adoc:525]:
```suggestion
Placeholders in sample code and commands represent values that the user must replace. Placeholders in example output can also represent other values that vary. Here are some rules for using placeholders:

- Use a descriptive name as the placeholder value. For example `<broker-id>`.
- Use lowercase characters.
- Separate words with a dash.
- Do not use possessives or instructions as values, such as `<replace-with or my-value>`.
```

**Comment by JakeSCahill** [comment]
> Maybe worth adding that it only works with code blocks and not code fences.

It should work with fences too as long as it’s a supported language.

**Comment by Deflaimun** [comment]
`Code fences` <- like this work too?

File: meta-docs/STYLE-GUIDE.adoc [hunk 2]
```diff
@@ -522,7 +522,13 @@ Avoid mentioning any future features or releases within the documentation. Such
...
 == Placeholders
 
-Placeholders in sample code and commands represent values that the user must replace. Placeholders in example output can also represent other values that vary. A placeholder has a descriptive name as its value. Always separate words with a dash. *Do not* use possessives or instructions as values, such as replace-with or my-value.
+Placeholders in sample code and commands represent values that the user must replace. Placeholders in example output can also represent other values that vary. Here are some rules for using placeholders:
+
+- Wrap placeholders in angled brackets like so `<placeholder>`.
+- Use a descriptive name as the placeholder value. For example `<broker-id>`.
+- Use lowercase characters.
+- Separate words with a dash.
+- Do not use possessives or instructions as values, such as `<replace-with or my-value>`.
```
Comment (Deflaimun) [inline meta-docs/STYLE-GUIDE.adoc:531]:
```suggestion
- Do not use possessives or instructions as values, such as `<replace-with> or <my-value>`.
```

File: meta-docs/STYLE-GUIDE.adoc [hunk 3]
```diff
@@ -535,9 +541,20 @@ When you use a placeholder in text or code, explain the placeholder the first ti
 rpk topic create <topic-name>
 ----
 
+Replace `<topic-name>` with the name that you want to give your new topic.
+
+=== Editable placeholders
+
 The Redpanda documentation UI includes a custom script that makes placeholder values in code blocks editable.
```
Comment (JakeSCahill) [inline meta-docs/STYLE-GUIDE.adoc:548]:
```suggestion
The Redpanda documentation UI includes a custom script that makes placeholder values in code blocks editable. Inline code fences are not editable.
```

**Pull request merged**